Tests/Async/ResolveAllCacheProcessorTest.php

```diff
@@ -0,0 +1,342 @@
+<?php
+namespace Liip\ImagineBundle\Tests\Async;
+
+use Enqueue\Client\ProducerInterface;
+use Enqueue\Consumption\Result;
+use Enqueue\Null\NullContext;
+use Enqueue\Null\NullMessage;
+use Liip\ImagineBundle\Async\ResolveAllCacheProcessor;
+use Liip\ImagineBundle\Async\Topics;
+use Liip\ImagineBundle\Imagine\Cache\CacheManager;
+use Liip\ImagineBundle\Imagine\Data\DataManager;
+use Liip\ImagineBundle\Imagine\Filter\FilterConfiguration;
+use Liip\ImagineBundle\Imagine\Filter\FilterManager;
+use Liip\ImagineBundle\Model\Binary;
+
+class ResolveAllCacheProcessorTest extends \PHPUnit_Framework_TestCase
+{
+    public static function setUpBeforeClass()
+    {
+        if (false == interface_exists('Enqueue\Psr\PsrProcessor')) {
+            throw new \PHPUnit_Framework_SkippedTestError('The enqueue bundle is not installed. Skip optional tests');
```

Comment:
I would use the parent class to handle this versus throwing this exception yourself:
```php
if (false == interface_exists('Enqueue\Psr\PsrProcessor')) {
  $this->markTestSkipped('The enqueue bundle is not installed. Skip optional tests');
}
```